prow/plank/controller_test.go

```diff
@@ -1335,6 +1410,41 @@ func TestSyncPendingJob(t *testing.T) {
 			ExpectedComplete: true,
 			ExpectedURL:      "homeless/error",
 		},
+		{
+			Name: "stale unschedulable prow job with specific podUnscheduledTimeout",
+			PJ: prowapi.ProwJob{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "homeless",
+					Namespace: "prowjobs",
+				},
+				Spec: prowapi.ProwJobSpec{
+					DecorationConfig: &prowapi.DecorationConfig{
+						PodUnscheduledTimeout: &metav1.Duration{Duration: 2 * time.Minute},
+					},
+				},
+				Status: prowapi.ProwJobStatus{
+					State:   prowapi.PendingState,
+					PodName: "homeless",
+				},
+			},
+			Pods: []v1.Pod{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:              "homeless",
+						Namespace:         "pods",
+						CreationTimestamp: metav1.Time{Time: time.Now().Add(-2*time.Minute - time.Second)},
```

Comment:
I am curious why an extra second needs to be subtracted here when the other two cases were able to be exactly the timeout

Reply:
I was just trying to create the same test as `stale unschedulable prow job` above, so I copied this test and just added the specific PodUnscheduledTimeout. I wanted to remain consistent with the existing code, but you are right, it is a bit unnecessary and the test will run successfully even without the one second